server/src/test/java/org/elasticsearch/cluster/coordination/NodeJoinTests.java

```diff
@@ -113,6 +113,8 @@ private static ClusterState initialState(DiscoveryNode localNode, long term, lon
                         .term(term)
                         .lastAcceptedConfiguration(config)
                         .lastCommittedConfiguration(config)
+                        .addVotingConfigExclusion(new CoordinationMetaData.VotingConfigExclusion(
+                            CoordinationMetaData.VotingConfigExclusion.MISSING_VALUE_MARKER, "knownNodeName"))
```

Comment:
I think it would be better to only add this in `testJoinUpdateVotingConfigExclusion` by taking the result of `initialState` and adjusting it. It will make the setup for that test a bit longer, but I don't think we should pollute other tests with this change.

Reply:
Done.